### DISCUSSION TITLE
Can I use GitHub logos in commercial video?

### DISCUSSION THREAD

**Discussion by eijitokyo** [community/community#22177] [New to GitHub]
<p>I’m making a video that introduces the R&amp;D facility of my client.<br>
It’s a kind of Interview video. The interviewee is talking about using GitHub in his development, so on.<br>
Can I use the GitHub logo and Octcat in this video?</p>

**Comment by darkRaspberry** [comment]
<p>I guess you need to read few lines stated in this <a href="https://github.com/logos" rel="noopener nofollow ugc">official website</a>. There are clearly mentioned what to do and what not with GitHub logo,</p>

**Comment by eijitokyo** [comment]
<p>I read official website.<br>
It read,</p>
<p>please contact us</p>
<ul>
<li>If you want to use these images in a video/mainstream media.</li>
</ul>
<p>So, I asked here.<br>
Thanks anyway.</p>

**Comment by darkRaspberry** [comment]
<p>ok wait till a staff contact you.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi, sorry I missed this - you’ll need to contact support directly if you have a question about using our logos and brand.</p>
<p>You can <a href="https://support.github.com/contact">open a ticket</a>. Select the copyright option to get the correct team!</p>
